generators/ci-cd/templates/.gitlab-ci.yml.ejs

```diff
@@ -152,18 +152,20 @@ before_script:
     <%_ } _%>
     - export MAVEN_USER_HOME=`pwd`/.maven
     <%_ if (!skipClient) { _%>
-        <%_ if (clientPackageManager === 'yarn') { _%>
+        <%_ if (!insideDocker) { _%>
+            <%_ if (clientPackageManager === 'yarn') { _%>
     - ./mvnw com.github.eirslett:frontend-maven-plugin:install-node-and-yarn -DnodeVersion=v<%= NODE_VERSION %> -DyarnVersion=v<%= YARN_VERSION %> -Dmaven.repo.local=$MAVEN_USER_HOME
-        <%_ } else if (clientPackageManager === 'npm') { _%>
+            <%_ } else if (clientPackageManager === 'npm') { _%>
     - ./mvnw com.github.eirslett:frontend-maven-plugin:install-node-and-npm -DnodeVersion=v<%= NODE_VERSION %> -DnpmVersion=<%= NPM_VERSION %> -Dmaven.repo.local=$MAVEN_USER_HOME
-        <%_ } _%>
+            <%_ } _%>
     - ./mvnw com.github.eirslett:frontend-maven-plugin:<%= clientPackageManager %> -Dmaven.repo.local=$MAVEN_USER_HOME
+        <%_ } _%>
     <%_ } _%>
 
 maven-compile:
     stage: build
     script:
-        - ./mvnw compile -Dmaven.repo.local=$MAVEN_USER_HOME
+        - ./mvnw compile -P-webpack -Dmaven.repo.local=$MAVEN_USER_HOME
```

Comment:
I think it will fail for microservice, as the `webpack` profile doesn't exist

Reply:
I have done a test and is passing. Since we just deactivate the profile is no problem. Even when a user is trying to activate a profile which not exists, maven compiles ok and it just raises a warning. Here is the output of the build when I explicitly try to activate profile _webpack_ which in a microservice is not existing. As I said just a warning:
<img width="914" alt="grafik" src="https://user-images.githubusercontent.com/4954023/66209732-5242ef00-e6b8-11e9-9dfc-c841c01ee938.png">